--- libcap-2.22/Make.Rules.orig	2011-07-25 04:18:23.000000000 +0200
+++ libcap-2.22/Make.Rules	2011-07-28 19:09:49.898308263 +0200
@@ -48,7 +48,7 @@
 IPATH += -fPIC -I$(topdir)/libcap/include -I$(KERNEL_HEADERS)
 
 CC := gcc
-CFLAGS := -O2 -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
+CFLAGS := $(OPT_CFLAGS) -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
 BUILD_CC := $(CC)
 BUILD_CFLAGS := $(CFLAGS) $(IPATH)
 AR := ar
@@ -59,7 +59,7 @@
         -Wstrict-prototypes -Wmissing-prototypes \
         -Wnested-externs -Winline -Wshadow
 LD=$(CC) -Wl,-x -shared
-LDFLAGS := #-g
+LDFLAGS := $(OPT_LDFLAGS) #-g
 
 SYSTEM_HEADERS = /usr/include
 INCS=$(topdir)/libcap/include/sys/capability.h
