--- libcap-2.20/libcap/include/linux/capability.h~	2011-01-19 06:00:55.000000000 +0100
+++ libcap-2.20/libcap/include/linux/capability.h	2011-01-20 23:34:36.261352242 +0100
@@ -362,7 +362,12 @@
 
 #define CAP_LAST_CAP         CAP_SYSLOG
 
-#define cap_valid(x) ((x) >= 0 && (x) <= CAP_LAST_CAP)
+/* Allow context manipulations */
+/* Allow changing context info on files */
+
+#define CAP_CONTEXT         63
+
+#define cap_valid(x) ((x) >= 0 && ((x) <= CAP_LAST_CAP || (x) == CAP_CONTEXT))
 
 /*
  * Bit location of each capability (used by user-space library and kernel)
