--- libcap-2.06/libcap/Makefile.orig	2008-02-07 19:58:32.000000000 +0100
+++ libcap-2.06/libcap/Makefile	2008-02-11 22:13:50.396611606 +0100
@@ -17,7 +17,6 @@
 MAJLIBNAME=$(LIBNAME).$(VERSION)
 MINLIBNAME=$(MAJLIBNAME).$(MINOR)
 GPERF_OUTPUT = _caps_output.gperf
-LDFLAGS+=-lattr
 
 all: $(MINLIBNAME) $(STALIBNAME)
 
@@ -59,10 +58,9 @@
 	install -m 0644 include/sys/capability.h $(INCDIR)/sys
 	mkdir -p -m 0755 $(LIBDIR)
 	install -m 0644 $(STALIBNAME) $(LIBDIR)/$(MINLIBNAME)
-	install -m 0644 $(MINLIBNAME) $(LIBDIR)/$(MINLIBNAME)
+	install -m 0755 $(MINLIBNAME) $(LIBDIR)/$(MINLIBNAME)
 	ln -sf $(MINLIBNAME) $(LIBDIR)/$(MAJLIBNAME)
 	ln -sf $(MAJLIBNAME) $(LIBDIR)/$(LIBNAME)
-	-/sbin/ldconfig
 
 clean:
 	$(LOCALCLEAN)
--- libcap-2.05/pam_cap/Makefile.orig	2008-01-22 01:16:58.000000000 +0100
+++ libcap-2.05/pam_cap/Makefile	2008-01-26 21:41:43.080250852 +0100
@@ -14,7 +14,7 @@
 	@echo "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
 
 pam_cap.so: pam_cap.o
-	$(LD) -o pam_cap.so $< $(LIBS)
+	$(LD) $(LDFLAGS) $(COPTFLAG) -o pam_cap.so $< $(LIBS)
 
 pam_cap.o: pam_cap.c
 	$(CC) $(CFLAGS) -c $< -o $@
--- libcap-2.05/progs/Makefile.orig	2008-01-22 01:16:58.000000000 +0100
+++ libcap-2.05/progs/Makefile	2008-01-26 21:40:49.333187982 +0100
@@ -9,7 +9,7 @@
 all: $(PROGS)
 
 $(PROGS): %: %.o
-	$(CC) --static $(COPTFLAG) $(LDFLAGS) -o $@ $< $(LIBS)
+	$(CC) $(COPTFLAG) $(LDFLAGS) -o $@ $< $(LIBS)
 
 %.o: %.c $(INCS)
 	$(CC) $(CFLAGS) -c $< -o $@
@@ -17,7 +17,7 @@
 install: all
 	mkdir -p -m 0755 $(SBINDIR)
 	for p in $(PROGS) ; do \
-		install -s -m 0755 $$p $(SBINDIR) ; \
+		install -m 0755 $$p $(SBINDIR) ; \
 	done
 
 clean:
