diff -urN libcap-1.92.orig/Make.Rules libcap-1.92/Make.Rules
--- libcap-1.92.orig/Make.Rules	Fri May 14 07:26:51 1999
+++ libcap-1.92/Make.Rules	Tue Apr 25 04:14:58 2000
@@ -44,10 +44,10 @@
 CC=gcc
 COPTFLAGS=-O2
 DEBUG=-g #-DDEBUG
-WARNINGS=-ansi -D_POSIX_SOURCE -Wall -Wwrite-strings \
+WARNINGS=-D_POSIX_SOURCE -Wall -Wwrite-strings \
         -Wpointer-arith -Wcast-qual -Wcast-align \
         -Wtraditional -Wstrict-prototypes -Wmissing-prototypes \
-        -Wnested-externs -Winline -Wshadow -pedantic
+        -Wnested-externs -Winline -Wshadow
 LD=ld
 LDFLAGS=-s #-g
 
