--- libcap-2.24/libcap/include/sys/capability.h~	2014-01-02 02:50:56.000000000 +0100
+++ libcap-2.24/libcap/include/sys/capability.h	2014-01-23 08:31:19.862345644 +0100
@@ -27,7 +27,6 @@
 #define __user
 #endif
 #include <linux/capability.h>
-#include <linux/xattr.h>
 
 /*
  * POSIX capability types
--- libcap-2.24/libcap/cap_file.c~	2013-12-24 18:35:13.000000000 +0100
+++ libcap-2.24/libcap/cap_file.c	2014-01-23 08:33:01.455678242 +0100
@@ -13,6 +13,7 @@
 #define XATTR_SECURITY_PREFIX "security."
 
 #include "libcap.h"
+#include <linux/xattr.h>
 
 #ifdef VFS_CAP_U32
 
